events/loginEvent.py

```diff
@@ -496,7 +496,10 @@ async def handle(web_handler: AsyncRequestHandler) -> tuple[str, bytes]:  # toke
 
         # Send to everyone our userpanel if we are not restricted or tournament
         if not osuToken.is_restricted(userToken["privileges"]):
-            await streamList.broadcast("main", await serverPackets.userPanel(userID))
+            await stream_messages.broadcast_data(
+                "main",
+                await serverPackets.userPanel(userID),
```

Comment:
unrelated to this PR but seeing a packet which is obviously doing more work than construction (the await) makes me scared lol

Reply:
lol yep, i knew about these from a while back. to be fixed at some point.